### DISCUSSION TITLE
Relation table between user and product

### DISCUSSION THREAD

**Discussion by alanaguiar01** [prisma/prisma#13553] [Q&A]
I have a question about relation tables and i would like to know how i make a relation the table user and product? I did a table post and inside the table how to put the information that the user will post? For exemple...

model User{
  Id:               Int    @id @default (autoincrement())
  name         String    @unique
  email          String    @unique
  password  String     @unique
  post Post[]
}

model Post {
  id         Int                 @id @default(autoincrement())
  user_id int 
  user User   @relation(fields:[user_id], references: [id])
  product_id int 
  user Product @relation(fields:[product_id], references: [id])
  title      String
  categories CategoriesOnPosts[]

  @@id([user_id, product_id])
}
model Product {
  Id.     Int
  name   String
  caracteristics String
  price: Decimal
  description    String
  category Category[]
}

model Category {
  id    Int                 @id @default(autoincrement())
  name  String
  posts CategoriesOnPosts[]
}

model CategoriesOnPosts {
  post       Post     @relation(fields: [postId], references: [id])
  postId     Int
  category   Category @relation(fields: [categoryId], references: [id])
  categoryId Int
  assignedAt DateTime @default(now())
  assignedBy String

  @@id([postId, categoryId])
}

Record:
User:[..., post[title, product[..., category []]]]

**Comment by nurul3101** [comment]
Hey @alanaguiar01 👋 
Can you please elaborate on what kind of relation do you want between `User` and `Product`?
Like One to One, One to Many etc

**Reply by vladi-stevanovic** [reply]
👋 Hello @alanaguiar01  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
